### ISSUE TITLE
pickle.load() failed for WK kernel

### ISSUE THREAD

**Issue by zhenzuo2** [theislab/cellrank#747]
When running 
```
import pickle
file_to_read  = open('wk.pickle', "rb")
wk = pickle.load(file_to_read)
```
I got the following errors

---------------------------------------------------------------------------
```
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_75099/2725592053.py in <module>
----> 1 wk = pickle.load(file_to_read)

/storage/singlecell/zz4/miniconda3/envs/mypython/lib/python3.8/site-packages/anndata/_core/views.py in __setitem__(self, idx, value)
     22 
     23     def __setitem__(self, idx: Any, value: Any):
---> 24         if self._view_args is None:
     25             super().__setitem__(idx, value)
     26         else:

AttributeError: 'DictView' object has no attribute '_view_args'
```

However, I did not get any error when loading anndata object. 
It will be very helpful if it can be loaded without recomputing it every time. Thank you.

**Comment by michalk8** [comment]
Thanks for spotting this, the issue was 1 `AnnData` view which can't be pickled, because we've also saved the `OTModel` from `wot` (wasn't necessary). Will be fixed in #748 and merged into `dev`; you can install it e.g. as:
```bash
pip install git+https://github.com/theislab/cellrank@dev
```

**Comment by michalk8** [comment]
closed via #748
